src/components/FormRenameStep.vue

```diff
@@ -44,20 +44,22 @@ import renameSchema from '@/assets/schemas/rename-step__schema.json';
 import WidgetInputText from './WidgetInputText.vue';
 import WidgetAutocomplete from '@/components/WidgetAutocomplete.vue';
 import { Getter, Mutation, State } from 'vuex-class';
+import { StepFormComponent } from '@/components/formlib';
 
 interface RenameStepConf {
   oldname: string;
   newname: string;
 }
 
-@Component({
+@StepFormComponent({
+  vqbstep: 'rename',
   name: 'form-rename-step',
```

Comment:
It should mirror the class name

Reply:
Just to be sure, Your're talking about `form-rename-step` vs. `step-rename-form`, right?

Reply:
Yep